docs/features/djangorestframework.md

```diff
@@ -36,15 +36,63 @@ validation.
 
 ````
 # models.py
+from ai_django_core.models import CommonInfo
+
 class MyOwnershipRelevantModel(CommonInfo):
     ...
 
 # serializers.py
+from ai_django_core.drf.serializers import CommonInfoSerializer
+
 class MyOwnershipRelevantModelSerializer(CommonInfoSerializer)
     class Meta:
         model = MyOwnershipRelevantModel
+        
+        # use __all__ to show all fields including the common ones
+        fields = '__all__' 
+
+        # or access them directly with 
+        # 'created_at', 'lastmodified_at',
+        # 'created_by', 'lastmodified_by'
+        # for instance
+        fields = ['id', 'name', 'created_at', 'lastmodified_at']
+        
+    # do your stuff here!
+    
+# access 'verbose' related commoninfo (meaning display name instead of ids)
+# can be acheive as such
+from ai_django_core.drf.serializers import CommonInfoSerializer
 
+class MyOwnershipRelevantModelSerializer(CommonInfoSerializer)
+    created_by = serializers.CharField(source='created_by.username', read_only=True)
```

Comment:
why do you declare them here again? thats not necessary and IMHO at little wrong because the created by is a foreign key and you are matching it with a username (CharField)